### DISCUSSION TITLE
Upstream DNS servers for certain domains

### DISCUSSION THREAD

**Discussion by gtmylab** [AdguardTeam/AdGuardHome#2856] [Q&A] [answered]
- [x] I am running the latest version
- [x] I checked the documentation and found no answer
- [x] I checked to make sure that this issue has not already been filed

My Adguard is running on a dedicated server and it had serves as a Primary Router DNS for us for the past 8 months.

Can a consideration be made to allow certain domain(s) to use Certain Upstream DNS servers ?.  A typical will be under #client you can add IP address, CIDR, MAC address or a special client ID to use certain Upstream DNS servers, can the same be done for Domain.

Since the lockdown, working remotely had made us always changing DNS server to access certain domains we use for work due to geo restriction. But when we change the DNS server for that Geo location it works. 

This is just a Feature request for consideration. Thank you

**Comment by ameshkov** [comment, marked as answer]
Please take a look here:
https://github.com/AdguardTeam/AdGuardHome/wiki/Configuration#upstreams-for-domains

Is it what you're looking for?

**Reply by gtmylab** [reply]
Thank You ! I never knew this was already available with AdGuard ! Exactly what I need, all now works well seamlessly. AdGuard  is really robust :)
